common/src/stack/command/stack/commands/add/pallet/__init__.py

```diff
@@ -198,6 +220,20 @@ def update_db(self, pallet_info, URL):
 			# Re-added the pallet. Update the URL.
 			self.db.execute('UPDATE rolls SET url=%s WHERE id=%s', (URL, rows[0][0]))
 
+	def perform_checksum(self, pallet_arg, iso_name):
+		if self.checksum.get(pallet_arg):
+			checksum_type, checksum_value = self.checksum.get(pallet_arg)
```

Comment:
Personal preference: I usually check for keys existance in dicts directly, so I don't have to double fetch the values. And once you know it is in the dict, you don't need get:
```
	if pallet_arg in self.checksum:
		checksum_type, checksum_value = self.checksum[pallet_arg]
```